homeassistant/components/asuswrt/__init__.py

```diff
@@ -1,120 +1 @@
-"""Support for ASUSWRT devices."""
```

Comment:
Moving from integration level config to platform level config is not allowed. See ADR7 and ADR10 for how integrations should be configured.

https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md

https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md